js/ui/map.js

```diff
@@ -417,8 +417,8 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
      * If the map's current zoom level is higher than the new maximum,
      * the map will zoom to the new maximum.
      *
-     * @param {number} maxZoom The maximum zoom level to set (0-20).
-     *   If`null` or `undefined` is provided, the function removes the current maximum zoom (sets it to 20).
+     * @param {number | null | undefined} maxZoom The maximum zoom level to set (0-20).
```

Comment:
That would be fine, too. I just copied from above for consistency. I will change both.